project.go

```diff
@@ -51,6 +51,9 @@ type Project struct {
 
 	Description string `jsonapi:"attr,description"`
 
+	// Optional: Indicates that the project is synced from HCP
```

Comment:
It's NBD, but none of these are an input attribute so no need to document as "Optional"